gtk/src/response_type.rs

```diff
@@ -137,3 +137,9 @@ impl ToValue for ResponseType {
         Self::static_type()
     }
 }
+
+impl From<ResponseType> for glib::Value {
+    fn from(t: ResponseType) -> Self {
+        t.to_value()
```

Comment:
```suggestion
        skip_init_asserts!();
        t.to_value()
```

or how the macro was called, I always forget :)